be/src/common/config.h

```diff
@@ -240,7 +240,8 @@ namespace config {
 
     // cumulative compaction policy: max delta file's size unit:B
     CONF_Int32(cumulative_compaction_check_interval_seconds, "10");
-    CONF_Int64(cumulative_compaction_num_singleton_deltas, "5");
```

Comment:
Does base compaction have this problem?

Reply:
No problem. It will not accumulate so much cumulative versions.